### DISCUSSION TITLE
[Config Support]: Storage Trouble

### DISCUSSION THREAD

**Discussion by jiiins** [blakeblackshear/frigate#15132] [Config Support]
### Describe the problem you are having

Even though there are still more than 500 GB left of free space, the logs say there is less than 1 hour left.

![image](https://github.com/user-attachments/assets/32b97bd9-1b8b-4d44-8eb2-89f0241a3fc9)

Eventually, it crashes with a "database or disk is full" log message and I have to restart.

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
.
```

### Relevant Frigate log output

```shell
info 2024-11-22 15:26:33 frigate.storage Less than 1 hour of recording space left, running storage maintenance...
info 2024-11-22 15:26:36 frigate.storage Cleaned up 25078.699999999986 MB of recordings

Repeated several times

I can't find the "database or disk is full" message as I have already restarted...
```

### Relevant go2rtc log output

```shell
.
```

### Operating system

Proxmox

### Install method

Proxmox via Docker

### docker-compose file or Docker CLI command

```yaml
.
```

### Object Detector

Coral

**Comment by NickM-27** [comment]
you have a lot of cameras, so realistically it is likely that it has less than 1 hour left of storage

**Reply by jiiins** [reply]
But that's a potential scenario, not the actual one. I have no issue with the preventive cleaning up, but why the crashes?

**Reply by NickM-27** [reply]
no information has been provided about crashes or anything of the sort, so realistically there is no way to answer that question
